projects/Mallard/ios/Podfile.lock

```diff
@@ -331,7 +331,7 @@ PODS:
     - React
   - RNCMaskedView (0.1.1):
     - React
-  - RNCPushNotificationIOS (1.2.2):
+  - RNCPushNotificationIOS (1.0.2):
```

Comment:
not sure this is deliberate

Reply:
Sorry, I had a merge conflict. This change must have happened during conflict resolution